UI-III-Flexbox/great-idea/index.html

```diff
@@ -0,0 +1,91 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Great Idea!</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+<div class="container">
+
+  <nav>
+    <div class="nav-bar">
+      <a href="services.html">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+
+      <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+    </div>
+  </nav>
+
+  <header>
+    <div class="head-left">
+      <h1>Innovation<br>
+          On<br>
+          Demand<br></h1>
+
+      <button type="button">Get Started</button>
+    </div>
+    <img src="img/header-img.png" alt="Image of a code snippet.">
+  </header>
+
+  <hr>
+
+  <section>
+    <div class="sec-top">
+      <h2>Features</h2>
+      
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+      <h2>About</h2>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+
+    <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+
+    <div class="sec-bot">
+      <h2>Services</h2>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+      <h2>Product</h2>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+      <h2>Vision</h2>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+  </section>
+
+  <hr>
+
+  <footer>
+    <address>
+      <h2>Contact</h2>
+      123 Way 456 Street<br>
+      Somewhere, USA<br><br>
+      <a href="tel:18888888888">1 (888) 888-8888</a> <br><br>
+      <a href="sales@greatidea.io">sales@greatidea.io</a>
```

Comment:
```suggestion
      <a href="mailto:sales@greatidea.io">sales@greatidea.io</a>
```
Similar to the phone number, you want 'mailto:EMAIL_ADDRESS' in your href so that this links to the email properly